README.md

```diff
@@ -5,13 +5,43 @@ The project introduces the Argo Rollout dashboard into the Argo CD Web UI.
 
 ![image](https://user-images.githubusercontent.com/426437/136460261-00d3dc31-ad20-4044-a7be-091803b8678f.png)
 
-# Quick Start
+## Install UI extension
 
-- Install Argo CD and Argo CD Extensions Controller: https://github.com/argoproj-labs/argocd-extensions
-- Create `argo-rollouts` extension in `argocd` namespace
+The UI extension needs to be installed by mounting the React component
+in Argo CD API server. This process can be automated by using the
+[argocd-extension-installer][1]. This installation method will run an
+init container that will download, extract and place the file in the
+correct location.
```

Comment:
This approach uses the [argocd-extension-installer](https://github.com/argoproj-labs/argocd-extension-installer) init container which runs during the startup of the argocd server.  
The init container downloads and extracts the JS file to `/tmp/extensions`. The argocd interface mounts the external JS file within the rollout resource.

Reply:
Updated, please take a look and re-review at your leisure.